appcenter-link-scripts/package.json

```diff
@@ -20,7 +20,7 @@
   "dependencies": {
     "debug": "4.1.1",
     "glob": "5.0.15",
-    "mkdirp": "0.5.1",
+    "mkdirp": "0.5.3",
```

Comment:
Hi @aphex3k, 

Thanks for your contribution. 

Can you please revert everything else and only keep this one line change? We have a dedicated internal workflow to test and release App Center RN SDK, so we will take care of the rest.

Reply:
Of course, that's what I had initially but the appcenter build failed so I thought I'd have to update the dependencies as well.

Reply:
Looks like our build now uses a private cocoapods repo for the testapp and thus fails when run from a fork, we will need to create a mirror PR to share the build status here.

Reply:
Anything that you want me to do about that or that I can do to help?